tests/unit/swagger/definitionsGeneration/definitions.spec.ts

```diff
@@ -834,7 +852,7 @@ describe('Definition generation', () => {
               example: 42,
               minimum: 42,
               maximum: 42,
-              default: '42',
+              default: 42,
```

Comment:
Now default types are ok, so we can use default values with @default jsdoc comment